xml/deploy_cephadm.xml

```diff
@@ -207,10 +197,20 @@
      Edit <filename>/etc/salt/minion</filename> and uncomment the following
      line:
     </para>
-<screen>#log_level_logfile: WARN</screen>
+<screen>#log_level_logfile: warning</screen>
     <para>
-     Change the <literal>WARN</literal> log level to <literal>INFO</literal>.
+     Change the <literal>warning</literal> log level to
+     <literal>info</literal>.
     </para>
+    <note>
+     <title><option>log_level_logfile</option> and <option>log_level</option></title>
+     <para>
+      While <option>log_level</option> controls which log messages will be
+      displayed on the screen, <option>log_level_logfile</option> controls
+      which log messages will be written to
+      <filename>/var/log/salt/master</filename>.
```

Comment:
I think we need `/var/log/salt/minion` here, since we are talking about the salt-minion daemon.

Reply:
fixed